packages/ag-charts-community/src/chart/interaction/keyBindings.ts

```diff
@@ -0,0 +1,51 @@
+type KeyBinding = { key: string; ctrlOrMeta?: boolean; shift?: boolean } | { code: string };
+
+type KeyActionName =
+    | 'arrowdown'
+    | 'arrowleft'
+    | 'arrowright'
+    | 'arrowup'
+    | 'undo'
+    | 'redo'
+    | 'submit'
+    | 'zoomin'
+    | 'zoomout';
+
+const KEY_BINDINGS: { [K in KeyActionName]: Readonly<KeyBinding>[] } = {
+    arrowdown: [{ code: 'ArrowDown' }],
+    arrowleft: [{ code: 'ArrowLeft' }],
+    arrowright: [{ code: 'ArrowRight' }],
+    arrowup: [{ code: 'ArrowUp' }],
+    redo: [
+        { key: 'y', ctrlOrMeta: true },
+        { key: 'z', ctrlOrMeta: true, shift: true },
+    ],
+    undo: [{ key: 'z', ctrlOrMeta: true }],
+    submit: [{ key: 'Enter' }, { code: 'Enter' }, { code: 'Space' }],
```

Comment:
👍 Nice, I like the encoding here.

This opens the door to adding a keyboard help dialog off the back of the config here too.